report/requirements.txt

```diff
@@ -1 +1,4 @@
-taskcluster==24.2.0
+# Switch to next release of Taskcluster once https://github.com/taskcluster/taskcluster/pull/2294 lands
+https://github.com/La0/taskcluster/archive/client-py-upload-artifact.zip#egg=taskcluster&subdirectory=clients/client-py
```

Comment:
We could actually wait for that before merging this, so we don't have to do this (and we don't have to do the changes in setup.py).

Reply:
Updated to 24.3.1